.github/workflows/lint-docs.yaml

```
@@ -17,7 +17,7 @@ jobs:
     if: github.repository == 'metal3-io/baremetal-operator'
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
+    - uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.3.0
```

```suggestion
    - uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
```

Bot bugs, fixing the tag manually.